assets/js/components/NotificationFromServer.js

```diff
@@ -104,9 +104,9 @@ function NotificationFromServer( {
 			ctaButton={ {
 				label: ctaLabel,
 				href: ctaURL,
-				target: ctaTarget,
 				onClick: onCTAClick,
 				dismissOptions,
+				external: ctaTarget === '_blank',
```

Comment:
I left a larger comment about this in my PR review… I don't think this is intuitive, but I don't think refactoring it is worth the effort/complications.

But I think adding some documentation/a comment here explaining that we do this because it's consistent with how notifications work and adding a new prop would be a bigger/needless task would clarify this for future reviewers/developers.

My immediate instinct was that this wasn't intuitive and should be improved upon, but once I dug into the code here I realised the level of effort needed and don't think it's worth it. Let's add something like this to warn others of wandering down the same road I went down 😆
```suggestion
				// In the case of notifications, other `target="_blank"` CTA URLs
				// are external, and adding a new, more explicit/clear `external`
				// prop would be a refactor of the service, JS, and PHP code all
				// for mildly more "intuitive" prop names/API.
				//
				// In this case it's not worth the refactor (as of writing this
				// logic), so we allow for any "open in a new window" URLs
				// to also be "external".
				external: ctaTarget === '_blank',
```